src/tuist.ts

```diff
@@ -0,0 +1,345 @@
+          options: [
+            {
+              name: ["--xcodeproj-path", "-p"],
+              description:
+                "Required. Path to the Xcode project whose build settings will be checked",
+            },
+            {
+              name: ["--target", "-t"],
+              description:
+                "Optional. The name of the target whose build settings will be checked. When not passed, it checks the build settings of the project",
+            },
+          ],
+        },
+        {
+          name: "list-targets",
+          description:
+            "When passed it uses xcframeworks (simulator and device) from the cache instead of frameworks (only simulator). Default is false",
+          options: [
+            {
+              name: ["--xcodeproj-path", "-p"],
+              description:
+                "Required. Path to the Xcode project whose build settings will be checked",
+            },
+          ],
+        },
+      ],
+    },
+    {
+      name: "scaffold",
+      description:
+        "Generates new project based on a template. For more info see https://docs.tuist.io/commands/scaffold USAGE: tuist scaffold <template> [--json] [--path <path>] <subcommand>",
+      args: {
+        name: "template",
+        description: "Name of template you want to use",
+      },
+      subcommands: [
+        {
+          name: "list",
+          description: "Lists available scaffold templates",
+        },
+      ],
+      options: [
+        {
+          name: ["--path", "-p"],
+          description:
+            "The path to the folder where the template will be generated (Default: Current directory)",
+        },
+        {
+          name: "--json",
+          description: "The output in JSON format",
+        },
+        {
+          name: "--name",
+          description: "The name of the generate project",
+        },
+        {
+          name: "--platform",
+          description: "The platform used by the project",
+        },
+      ],
+    },
+    {
+      name: "test",
+      description:
+        "Tests a project. For more info see https://docs.tuist.io/commands/test USAGE: tuist test [<scheme>] [--clean] [--path <path>] [--device <device>] [--os <os>] [--configuration <configuration>] [--skip-ui-tests] [--result-bundle-path <result-bundle-path>] [--retry-count <retry-count>]",
+      args: {
+        name: "scheme",
+        description:
+          "The scheme to be tested. By default it tests all the testable targets of the project in the current directory",
+      },
+      options: [
+        {
+          name: ["--path", "-p"],
+          description:
+            "The path to the directory that contains the projectto be tested",
+        },
+        {
+          name: ["--clean", "-c"],
+          description: "When passed, it cleans the project before testing it",
+        },
+        {
+          name: ["--device", "-d"],
+          description: "Test on a specific device",
```

Comment:
A generator would be even nicer here too
```suggestion
          description: "Test on a specific device",
          args: {
            name: "device",
          },
```